modules/ixBidAdapter.js

```diff
@@ -591,7 +591,7 @@ function buildRequest(validBidRequests, bidderRequest, impressions, version) {
     }
 
     if (bidderRequest.refererInfo) {
-      r.site.page = bidderRequest.refererInfo.referer;
+      r.site.page = config.getConfig('pageUrl') || bidderRequest.refererInfo.referer;
```

Comment:
do you want to sanity check values?

Reply:
Good point. I've made changes to set `site.page` only if pageUrl is valid.